### DISCUSSION TITLE
My fellow Kindred _ Miei cari fratelli - Maddecchè!!!

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#45] [Generale]
una divagazione su una frase che sembra nulla ma invece:

La frase in inglese dell'intro con il principe che fa la sua entrata. "My fellow Kindred".

come andrebbe tradotta veramente?
sappiamo che ogni traduzione ha proposto "Miei cari Fratelli".
ma non è mica per niente giusta!

da dove salta fuori quel "cari"

analizzando i termini e le relative possibili traduzioni piu consone addatte al caso.

Kindred = Parente, affine, consanguinei.

Ma Kindred ormai è stabilito che è Fratelli nel senso di fratellanza dei vampiri. ok, ci stà.

Quindi siamo a "Miei fratelli (cosa?)"

per fellow ci dobbiamo mettere "cari"? ma non è mica corretto!

sappiamo cosa significa esattamente "fellow". dobbiamo solo scegliere il termine italiano piu appropriato al caso.

fellow = compagni, colleghi, membri, seguaci.

facciamo qualche prova, vediamo come suona:

1) - Miei compagni Fratelli.
2) - Miei Fratelli colleghi
3) - Miei Fratelli membri
4) - Miei Fratelli seguaci.

Tenendo presente che i presenti nel teatro fanno parte di un elitè, non tutti i vampiri sono riuniti al processo.

non so perchè ma sono combattuto fra la terza e la quarta, Miei Fratelli membri o Miei Fratelli seguaci.?

mi piace "seguaci", mi sa di fratelli di sangue etc...

ma "Miei Fratelli membri" mi sembra il più consono, appunto membri di quell'elitè, etc...

che ne dite della divagazione traduttoria?

date un opinione per favore, grazie

**Comment by NarakuITA** [comment]
Mi piace: Miei Fratelli seguaci, anche perché da quello che ho capito lui è un vampiro importante. Dunque il termine seguace è più adatto per chi sta sotto di lui, nella scala gerarchica.

**Reply by crazyLillo** [reply]
E "Fratelli seguaci" sia.

anchio opto per questa soluzione, seguaci, per il principe ci sta a pennello.

Quindi sarà "Miei fratelli seguaci".

**Comment by tesivo** [comment]
Concordo anche io

**Comment by Root78** [comment]
Arrivo tardi, ma forse seguaci non è il miglior termine. Tanti all'interno del teatro non sono con la Camarilla, come Nines, Jack, Isaac ecc. Sarebbe un insulto essere chiamati seguaci per gli anarchici, e LaCroix vuole salvarsi il culo politicamente (per quello salva il protagonista) 

Si potrebbe eliminare il problema usando "Gentili Fratelli" "

**Reply by crazyLillo** [reply]
Felice di sentire altre voci (virtuali...)

Cercherò di essere breve, ma scusa sono logorroico di natura...

Lo sai che ha i ragione! alla fine giu in basso ti dico perchè.


Mi sento un po solo a dire la verità, ma capisco che ognuno disperda piu energie in cose che lo attirano di piu in una scala gerarchica virtuale tutta personale.
A parte chi per studio e chi per lavoro...ma va bene così,  (è la vita...)

Sono arrivato qui per una serie di fortunate circostanze, mi ci sono soffermato perchè la linea che tesivo si è impostato mi è piacuta, vuoi per il fatto che lui punta a una traduzione il più completa possibile innanzitutto,
e poi sopratutto perchè punta alla "Correttezza" di essa. (cosa molto importante dal mio punto di vista, e come afffermato più volte da tesivo). La linea è quella.

è la traduzione "Interpretativa", che non vogliamo utilizzare, o almeno rinunciarci il più possibile, e utilizzarla solo in quei punti dove risulta veramente impossibile proporre una traduzione efficace.
Tipo quando una frase allude a cose o fatti che conoscono solo li nel distretto di Los Angeles o in America in generale e che non hanno corrispettivo nell'idioma italico.
Ma è lì che subentro io avendo conoscenze abbastanza approfondite dei luoghi, delle persone e dei fatti, l'ho già dimostrato rivelando retroscena sconosciuti in alcune frasi.

Ma non voglio apparire "saccente" e arrogante, l'inglese è pure la lingua che conosco meno, io sono per lo piu esperto di lingue orientali antiche, crittografia e decrittografia e tante altre cose e bla bla bla.
Tutta roba inutile ai fini del gioco vampire.

ho solo detto da qualche parte qui, che ho vissuto da quelle parti abbastanza da poter aiutare nella traduzione, (penso che lo sto dimostrando alla grande, come si dice...non sto qui per pettinare bambole. )

...e qui che ce ne vorrebbero tanti altri come te a discuterne, perchè sarebbe salutare ai fini del gioco e di quello che stiamo facendo. Purtroppo siamo rimasti in pochi a volerlo e ci dobbiamo arrangiare,
i tuoi pareri, quindi saranno sempre ben accetti, non esitare a intervenire su tutte quelle modifiche migliorative che sto proponendo.
Ahime!se ci fossero tanti altri traduttori qui a discuterne, sarebbe molto proficuo, perchè ognuno potrebbe tirare fuori dal suo bagaglio personale il termine più azzeccato in assoluto.
Purtroppo siamo rimasti in pochi con la passione per la traduzione di questo gioco, ma sono convinto che fra 10 anni questo videogame sarà ancora giocato da qualche ragazzo, magari tutto completamente in italiano, 

comprese mods e tutto quello che gira intorno a Vampire Bloodlines.


Allora

Avevo scritto questo nel primo messaggio:

(((La frase in inglese dell'intro con il principe che fa la sua entrata. "My fellow Kindred".
come andrebbe tradotta veramente?
sappiamo che ogni traduzione ha proposto "Miei cari Fratelli".
ma non è mica per niente giusta!
da dove salta fuori quel "cari".)))


Tu invece proponi :

(((Si potrebbe eliminare il problema usando "Gentili Fratelli" ")))


Ma a questo punto "anche "Gentili", rientra nell' Intepretatorio come "cari".

Non devi farne una questione personale, per carità, non fraintendermi!
Ma è per la linea che appunto stiamo seguendo con tesivo.
Altrimenti così andava benissimo anche la primissima traduzione che fu fatta tanto tempo fa.
Chi ci ha giocato a quel tempo non conoscendo l'inglese, è riuscito comunque a finirlo con i sottotitoli proposti.
la giocabilità c'era comunque, ma come ripeto è la correttezza dell'idea originale che ci preme.

c'è molto da lavorare sulla traduzione attualmente disponibile, che mi dispiace dirlo è strapiena di stranezze traduttorie.
Non conosco gli autori, non me ne volessero, non è niente di personale, non me ne viene in tasca nulla.
non è cattiveria e non è un attacco diretto, dato che capisco che qualcuno potrebbe risentirsene.

Potrò risultare anche antipatico, ma io dico quello che vedo e che sopratutto vedono tutti, avendo a disposizione tutte le traduzioni fatte fin'ora,
e i testi originali in inglese. è un lavoraccio quello che sto facendo, revisionare tutto!

un esempio di tanti di quello che dico :

Il  file "tutorial security guard.dlg" in Unofficial_Patch\dlg\downtown la.
( i dialoghi con la guardia di sicurezza nel sotteraneo per intenderci)

frase originale inglese della linea 25.

{	25	}{	With me at your side you may have a chance.


andate a vedere qui da voi nella traduzione italiana che avete a disposizione, è tradotta così!:
{	25	}{	Forse. E forse posso divertirmi a prenderti a calci...


io ho dovuto corregerla così, sempre per quella correttezza che andiamo cercando io e tesivo
{	25	}{	Con me al tuo fianco potresti avere una chance.	


Tutto questo ci riporta a punto e daccapo qui :

Ormai stabilito che ai fini del gioco Kindred = Fratello\i. Anche se Kindred = Parente, affine, consanguinei.

Quindi siamo a "Miei fratelli (cosa?)"


per "fellow" cosa mettiamo? c'è lo inventiamo? ( io sarei tentato con "miei fottuti fratelli"...che per  LaCroix ci starebbe bene...)

scherzi a parte, sappiamo cosa significa esattamente "fellow". dobbiamo solo scegliere il termine italiano piu appropriato al caso.

ma allora facciamolo!!!

quelli più consoni al nostro caso sono questi

fellow = compagni, colleghi, membri, seguaci. e guarda un pò si potrebbe tradurre anche con "Fratelli".

facciamo qualche prova reale, vediamo come suona:

        Miei compagni Fratelli.
        Miei Fratelli colleghi
        Miei Fratelli membri
        Miei Fratelli seguaci.

se capovolgiamo i termini si puo arrivare anche a "Miei Fratelli Consanguinei"

avevo scritto che "Fratelli Seguaci mi piaceva ma che consideravo "Miei Fratelli membri"  il più consono,
Tenendo presente che i presenti nel teatro fanno parte di un elitè,  appunto membri di quell'elitè, etc..
perchè non tutti i vampiri hanno il permesso di presenziare al processo.
o almeno questa è la mia ipotesi. 
avevo chiesto parere è la scelta è capitata su "seguaci"

Ma come fai giustamente notare (è qui hai ragione anche tu) quello che hai spiegato non fa una piega e ci sta bene.
quindi andando per esclusione di quello che abbiamo a disposizione, 
ok, scartiamo "seguaci". ma scartiamo anche "colleghi e compagni". per lo stesso discorso che hai fatto.

Ci rimane solo "Membri" appunto membri della società vampirica a cui tutti i vampiri appartengono, cosa su cui non ci piove.
 Indipendentemente dalle fazioni, (anche se metaforicamente pregano "un chiodo", "una vite" o " un bullone", con le loro idee politiche, sono tutti membri della società vampirica.

che ne dite? ci può stare, accontentiamo tutti?

**Reply by NarakuITA** [reply]
Non mi dispiace **Miei Fratelli Consanguinei**.  Seguace non credo sia proprio azzeccato, visto che come detto da Root78, ci sono degli anarchici tra i vampiri. Dunque di sicuro non sono dei "Seguaci" veri e propri del principe che parla sul teatro.

**Reply by Root78** [reply]
Non vederlo come una critica Lillo, ma conosco abbastanza bene il gioco visto che ci ho giocato un sacco di volte.

Concordo che la traduzione sia veramente meh in certi punti, e a volte l unico modo per controllarla e non perdere pezzi per strada è testare come risulta in game.

Comunque detto ciò "conseanguinei" è molto pesante come termine, ma ci sta con il fatto che LaCroix è un Ventrue spocchioso.

In generale comunque anche se si elimina "fellow" non lo vedo come un problema grave, molto spesso le frasi devono essere modificate o interpretate.

**Reply by NarakuITA** [reply]
Concordo, è bene cercare di tradurre il più fedelmente possibile un testo originale, ma bisogna sempre tenere a mente la nostra grammatica, molto più elaborata e complessa di quella inglese. Dunque bisogna, si tradurre "bene", ma tenere  mente la "leggibilità fluida" del testo tradotto nella nostra lingua. 
Dunque dire anche "Miei cari" non è sbagliato, perché la frase è più fluida e leggibile. Ovviamente se si vuole essere più fedeli, il "cari" non va messo. Ma però, parere personale, leggere **Miei Cari Fratelli Consanguinei** è più leggibile.

**Reply by crazyLillo** [reply]
io non le vedo come critiche, ma consigli, infatti ho dato ragione a Root78 sul fatto di Lacroix.
Ma la traduzione in se stessa è un altra cosa

se sono critiche io le accetto, anzi non chiedo altro, servono allo scopo che ci siamo prefissati!
Però eliminare termini solo perchè "scomodi" da tradurre non è l'approccio giusto, sarebbe riduttivo, pressapochistico e anche controproducente.

Ricordo che siamo qui per migliorare la traduzione...

Sennò, si finirebbe di fare lo stesso lavoro fatto finora da altri. A che prò? 
Allora facciamoci andare bene la traduzione esistente e buonanotte!

Premetto che le traduzioni interpretative non vanno daccordo con il mio modo di lavorare. Il pane è pane e il vino è vino, punto!

se vai dal panettiere gli dici "dammi un kilo di pane" (magari specificando il tipo).
non gli chiedi " mi dia un kilo di quella massa che impastate con acqua e farina e fate lievitare e poi infornate"!

dal vinaio gli chiedi un litro di vino.
non gli chiedi " mi dia un litro di quel liquido che fanno spremendo i chicchi di un frutto chiamato uva che poi fate fermentare..."

ohh, certo ti potranno anche capire, ma perlomeno ti guardano strano...

il senso è che quando traduco un termine, se è di dubbia interpretazione, vado a vedermi tutti i termini possibili immaginabili, è uso il più consono al contesto, 
per avere una traduzione più conforme possibile all'originale. Questo è l'unico modo di tradurre che so applicare, ed è l'unico applicabile se vuoi fare le cose per bene.

immaginatevi di dover tradurre un testo molto importante o addirittura esagerando, di un qualche segreto industriale.
là, se non sei preciso più di un orologio svizzero...

lo sò, lo sò, è solo un videogame...ok. Ma lo vogliamo fare bene o vogliamo aggiungerci alla lista dell'ennesima traduzione di un videogame fatta con i piedi?
di esempi in giro c'è ne sono a bizzeffe...



root78 dice: 

 (((molto spesso le frasi devono essere modificate o interpretate.)))

un vero tradutttore non lo fa mai, è controproducente per la sua carriera, analizzando il lavoro svolto, diranno " questo non va bene, lavora troppo alla carlona, prendiamone un altro..."
...e i soldi vanno a finire nelle tasche di qualcun'altro...

modificare o iterpretare ci potrebbe stare in casi specifici di lingue veramente ostiche, antiche e quant'altro.
ma non è proprio questo il caso.

ma veniamo al punto, analizziamo la questione seriamente :

 "Miei Fratelli Consanguinei" esce fuori da 2 combinazioni, ovvero deriva dal fatto che:

Kindred = Parente\i, Affine\i, Consanguineo\i. (Fratelli, pensando ad associazioni religiose e non).

Fellow = Compagni, Colleghi, Membri, Seguaci, e anche "Fratelli".

in questo caso "Fellow", sarebbe "Fratelli" e "Kindred, sarebbe "Consanguinei" .

Con queste premesse, andando per esclusione, "Miei Fratelli Consanguinei", è l'unica che si può formare che abbia un "senso" con il contesto, 
purtroppo il "senso" così, non ha molta forza, non dico che sia sbagliato, ma non ci sta proprio azzeccatissimo.

Dobbiamo anche considerare il fatto che ormai "Kindred =Fratelli", nel senso di fratellanza vampirica appartenenti appunto alla società vampirica.
"Kindred =Fratelli" in questo caso ci sta troppo azzecato e non si cambierà mai.
(immaginatevi a ritradurre kindred per centinaia di volte i tutti i dialoghi...), Io non lo faccio!

Quindi stiamo ancora qui :

" Miei Fratelli (cosa?) "

" Miei (cosa?) Fratelli "

Bisogna per forza inserire il termine tradotto di "fellow".

quindi

Fellow = Compagni, Colleghi, Membri, Seguaci, e anche "Fratelli".

le possibili combinazioni sono solo queste:

Miei Fratelli Fratelli? (no, scartata in partenza)

Miei compagni Fratelli.
Miei Fratelli colleghi
Miei Fratelli membri
Miei Fratelli seguaci.

è vero, "seguaci", anche se un pò scenografica, non va proprio benissimo per il discorso sensato di root78.

"compagni" ? -  LaCroix è comunista?, LaCroix è in riunione scolastica?  per lo stesso discorso di root78, non va tanto bene.

ci rimangono solo :

Miei Fratelli colleghi
Miei Fratelli membri

"Miei Fratelli colleghi" (dovrebbe stare per fratelli "vampiri"), allora trasformiamolo in "Miei Fratelli Vampiri".  (non è un licenza poetica)
colleghi, mi sa troppo di collega d'ufficio etc...

"Miei Fratelli membri", appunto membri della società vampirica a cui tutti i vampiri appartengono.

Di scelte possibili e sopratutto plausibili ne sono rimaste solo 2 :

"Miei Fratelli colleghi" o se vi piace e prendendosi la licenza "Miei Fratelli Vampiri", appunto perchè colleghi di che? - Vampiri!
Ma "Kindred" = "Fratelli", stabilisce già la fratellanza vampirica (colleghi...). Quindi non è che mi ispira tantissimo perchè è ridondante.

Il termine più consono rimasto?

dopo tutte le premesse, "Miei Fratelli membri", rimane il termine più consono in assoluto! (quello che ho indicato fin da subito)

Non c'è un altro modo, se non stravolgendo i termini anglofoni, ma poi ritorniamo al discorso iniziale però.

quindi stiamo ancora al punto di cosa veramente dice Lacroix e non di quello che potrebbe dire o potrebbe significare.

ma io non voglio imporre nulla, mi attengo solo a quello che leggo, a me non viene in tasca nulla.

**Reply by Root78** [reply]
Come termine non mi sembra molto fluido onestamente, ma se è il più accurato nei confronti dell'originale allora ok. Oppure miei cari fratelli consanguinei come ha detto Naraku.

**Reply by crazyLillo** [reply]
Bhe! io ho ampiamente spiegato tutto l'andazzo traduttorio su questa frase.

vediamo che dice tesivo a questo punto

**Reply by tesivo** [reply]
Volendo è possibile tradurrei due termini anche nei seguenti modi:
fellow: amati, vicini, gentili
Kindred: Amici, Simili, Affini

Proporrei però di trovare un sistema più comodo per elencare le varie scelte e quindi passare ai voti. Potrebbe andare bene un sondaggio, ma non so se sulla frase o sulle parole, perchè nel primo caso le combinazioni possono essere davvero troppe da elencare

**Comment by crazyLillo** [comment]
Bhe! qui bisogna innanzitutto analizzare il contesto esatto, (come per i barboni...).

Riagganciandomi al discorso fatto da Root78,

Ho trovato in giro questo qui sotto (non l'ho scritto io... vengono dagli autori del gioco)
Voglio che l'attenzione vada ai pezzi di frase che io riporto in MAIUSCOLO.

Brujah
(((Considerati violenti e rissosi, la loro irruenza li porta ad avere dei fortissimi ideali di libertà, e per questo la maggior parte di loro 
fa parte della setta degli ANARCHICI. TUTTAVIA,  ALCUNI SONO AFFILIATI ALLA CAMARILLA.)))

Camarilla
(((La Camarilla può essere considerata la setta più "civilizzata", contrapposta alla barbarie del Sabbat. 
Nata durante il Medioevo con lo scopo di preservare la Masquerade,
essa assume che TUTTI I VAMPIRI NE FACCIANO PARTE, VOLENTI O NOLENTI!)))

Quindi il discorso di Root78, poteva andar bene fino a un certo punto.

Ma questo è quanto:

ALCUNI ANARCHICI SONO AFFILIATI ALLA CAMARILLA.

TUTTI I VAMPIRI FANNO PARTE DELLA CAMARILLA, VOLENTI O NOLENTI!


Quindi dalla mia anamnesi risulta che il termine più appropriato è proprio quello che ho indicato già da tempo :

Dato che non si può assolutamente cambiare "kindred" in nessun altro modo se non "Fratelli". (anche per gli anglofoni "kindred = Brother (Fratelli).
 
Quindi :

My fellow kindred = Mie Fratelli Membri.

Membri della camarilla (volenti o nolenti)

io punto all'esattezza e su questo non mi smuovo, poi se vi piacciono traduzioni di comodo, chi sono io per impedirlo?
